bonfire/utils.py

```diff
@@ -13,7 +13,7 @@
 from pathlib import Path
 from urllib.parse import urlparse
 
-import pkg_resources
+from importlib.metadata import version, PackageNotFoundError
```

Comment:
It looks like from a quick google search that using `pkg_resources` is partially deprecated, was this the reason for the change or was there another reason?

Reply:
Yep, exactly, I read about it while reading: https://setuptools.pypa.io/en/latest/pkg_resources.html#package-discovery-and-resource-access-using-pkg-resources